src/Raven.Studio/typescript/viewmodels/database/tasks/editReplicationHubTask.ts

```diff
@@ -139,6 +148,29 @@ class editReplicationHubTask extends viewModelBase {
     }
 
     private initObservables() {
+        this.visibleReplicationAccessItems = ko.pureComputed(() => {
+            let itemsToShow = this.allReplicationAccessItems();
+
+            const filter = this.filterText();
+            if (filter) {
+                const upperFilter = filter.toUpperCase();
+                itemsToShow = itemsToShow.filter(x => x.replicationAccessName().toLowerCase().includes(upperFilter) ||
```

Comment:
can we avoid recomputing filtered items on every load?

Reply:
done